### ISSUE TITLE
Terminate after last window closed, please

### ISSUE THREAD

**Issue by artginzburg** [ThatsJustCheesy/defaults-edit#1]
Hey, it would be cool to add this to AppDelegate. I just don't really think that this app needs to be opened in background. And I want to close it with a simple red button.
```
func applicationShouldTerminateAfterLastWindowClosed(_ sender: NSApplication) -> Bool {
        return true
}
```

Another cool option is to open a domain selection window when some domain is closed, and fully close the app after selection window is closed

**Comment by ThatsJustCheesy** [comment]
Hey, sorry I missed this earlier—but it's now implemented in v0.2!

As for the second suggestion: I don't think closing the window should open a new one, even if it had a domain open. The average user would likely find that behaviour confusing.

**Comment by aidan-gibson** [comment]
Still have this issue (closing last window doesn't close app). Version 0.4.0. 

(Also of note, re-clicking on the app icon after closing a window does not open a new window)

**Comment by ThatsJustCheesy** [comment]
Hi @aidan-gibson, yeah, I actually reverted this change after adding `defaults-edit://` URL handling so that the app wouldn't close and open again for each URL you click.

It should open a new window on "relaunch", though. Tracked in #4

btw, I haven't worked on this in a couple years 😅 so while these (#3, #4) are simple bugs, it may be a while before I get to them. I do eventually™ want to finish this app (i.e., address the [todo items in the README](https://github.com/ThatsJustCheesy/defaults-edit#todos)), but am currently deep in other things. (Of course, if you want to try, I would happily take a PR that fixes #3 or #4)
